### PR TITLE
look only for ping or pong as whole words

### PR DESCRIPTION
**Pull request by westmaas**

### CODE REVIEW

**Comment by westmaas** [comment]
keeping the trigger detection strict

**Comment by cohitre** [comment]
we should check for word boundaries (using `\b`) since often comments will not be just the word "ping"

File: server.rb [hunk 1]
```diff
@@ -46,7 +46,7 @@ def review_label!(repository, issue)
 end
 
 def comment_created_message(redis, repository, issue, comment)
-  if comment.matches?(/p(i|o)ng/i)
+  if comment.matches?(/^p(i|o)ng$/i)
```
Comment (ironcamel) [inline server.rb]: `(i|o)` is the noob way to do `[io]`. come on gabe, stop wasting characters.

**Comment by cohitre** [comment]
lgtm

**Comment by ironcamel** [comment]
yes, it is important to notifiy people when my dog is rip.ping a major fart

**Pull request merged**